app/src/main/java/com/example/henzoshimada/feeltrip/EditMoodActivity.java

```diff
@@ -32,6 +32,21 @@
     DateFormat formatDateTime = DateFormat.getDateInstance();
     private Calendar dateTime = Calendar.getInstance();
 
+    private String emotionalState;
+    private String socialSit;
+
+    // The following are constants for emotion based emoji
+    public static final int angry = 0x1F624;
+    public static final int confused = 0x1F635;
+    public static final int disgusted = 0x1F623; //might change this one...
+    public static final int fearful = 0x1F628;
+    public static final int happy = 0x1F60A;
+    public static final int sad = 0x1F622;
+    public static final int shameful = 0x1F61E;
+    public static final int cool = 0x1F60E;
```

Comment:
These are UNICODE integers that represent every single one of the emotions we have. This list makes adding new emotions a trivial task, should we want to do that.